pkg/services/login/login.go

```diff
@@ -112,16 +108,14 @@ func (ls *LoginService) UpsertUser(cmd *models.UpsertUserCommand) error {
 		}
 	}
 
-	err = ls.Bus.Dispatch(&models.SyncTeamsCommand{
+	if err := ls.Bus.Dispatch(&models.SyncTeamsCommand{
```

Comment:
Nit. Not sure this reads better than before. I suggest moving the `if and condition` below `ls.Bus.Dispatch(&models.SyncTeamsCommand{...`

Reply:
@marefr OK I can make the change; the idea was to scope `err` locally to the `if` statement, it helps with avoiding usage of dangling `err`s.